pkg/pillar/containerd/containerd.go

```diff
@@ -791,8 +791,20 @@ func getSavedImageInfo(containerPath string) (ocispec.Image, error) {
 		logrus.Errorf("getSavedImageInfo: could not read image config file: %v", err)
 		return image, err
 	}
+	if len(data) == 0 {
+		files, err := os.ReadDir(filepath.Join(containerPath, "rootfs"))
+		if err != nil {
+			logrus.Error("rootfs directory also not present")
+		}
+
+		fileNames := ""
```

Comment:
@europaul what is the rationale to list the whole rootfs directory? This can cause a huge error message (and time to processing) for containers with high volume of files in their rootfs.... in this error case (empty OCI file), wouldn't be enough just to tell if the rootfs directory is also present or not? but without listing the whole directory....